scripts/check-migrations.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+from shutil import ExecError
+
+"""
+This script is meant to be run in CI to check that migrations changes are
+not coupled with other changes.
+"""
+
+
+class CoupledMigrations(Exception):
+    pass
+
+
+def _get_migration_changes(workdir: str = ".", to: str = "origin/master") -> str:
```

Comment:
Nit: Since you have default parameters defined in main itself, you can remove the defaults from all other places.

Reply:
removed